test/extended/util/pods.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func WaitForNoPodsAvailable(oc *CLI) error {
+	return wait.Poll(200*time.Millisecond, 3*time.Minute, func() (bool, error) {
+		//ep, err := oc.KubeClient().CoreV1().Endpoints(oc.Namespace()).Get(serviceName)
```

Comment:
delete?

Reply:
That was in the original method that I re-homed, but i'll go ahead and remove it.